core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -60,6 +60,39 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
       "--add-config", "security.inter.broker.protocol=PLAINTEXT"))
   }
 
+  @Test
+  def shouldExitWithNonZeroStatusOnZkCommandWithTopicsEntity(): Unit = {
+    assertNonZeroStatusExit(Array(
+      "--zookeeper", zkConnect,
+      "--entity-type", "topics",
+      "--describe"))
+  }
+
+  @Test
+  def shouldExitWithNonZeroStatusOnZkCommandWithClientsEntity(): Unit = {
+    assertNonZeroStatusExit(Array(
+      "--zookeeper", zkConnect,
+      "--entity-type", "clients",
+      "--describe"))
+  }
+
+  @Test
+  def shouldExitWithNonZeroStatusOnZkCommandWithIpsEntity(): Unit = {
+    assertNonZeroStatusExit(Array(
+      "--zookeeper", zkConnect,
+      "--entity-type", "ips",
+      "--describe"))
+  }
+
+  @Test
+  def shouldExitWithNonZeroStatusAlterUserQuotaWithoutEntityName(): Unit = {
```

Comment:
I was thinking this test should be exactly like the `clients` entity case, why did we make it different?

Reply:
> I was thinking this test should be exactly like the clients entity case, why did we make it different?

Do you mean this test case?
```java
@Test
  def shouldExitWithNonZeroStatusOnZkCommandWithClientsEntity(): Unit = {
    assertNonZeroStatusExit(Array(
      "--zookeeper", zkConnect,
      "--entity-type", "clients",
      "--describe"))
  }
```
If so, it's because `users` entity can still accept to use ZK because we need it to create/update the SCRAM credentials. 
If this is not what you meant, please kindly let me know. Thank you. :)

Reply:
But the CLI can verify if you are trying to update SCRAM credentials vs updating quotas. We should not allow the latter.

Reply:
Aha, I know what you mean now. Let me update it. Thanks 😊